tests/Jellyfin.MediaEncoding.Tests/Probing/ProbeResultNormalizerTests.cs

```diff
@@ -71,5 +72,32 @@ public void GetMediaInfo_MusicVideo_Success()
             Assert.True(res.PremiereDate.HasValue);
             Assert.Equal(DateTime.Parse("2021-01-01T00:00Z", DateTimeFormatInfo.CurrentInfo).ToUniversalTime(), res.PremiereDate);
         }
+
+        [Fact]
+        public void GetMediaInfo_Music_Success()
+        {
+            var bytes = File.ReadAllBytes("Test Data/Probing/music_metadata.json");
+            var internalMediaInfoResult = JsonSerializer.Deserialize<InternalMediaInfoResult>(bytes, _jsonOptions);
+            MediaInfo res = _probeResultNormalizer.GetMediaInfo(internalMediaInfoResult, null, true, "Test Data/Probing/music.flac", MediaProtocol.File);
+
+            Assert.Equal("UP NO MORE", res.Name);
+            Assert.Single(res.Artists);
+            Assert.Equal("TWICE", res.Artists[0]);
+            Assert.Equal("Eyes wide open", res.Album);
+            Assert.Equal(2020, res.ProductionYear);
+            Assert.True(res.PremiereDate.HasValue);
+            Assert.Equal(DateTime.Parse("2020-10-26T00:00Z", DateTimeFormatInfo.CurrentInfo).ToUniversalTime(), res.PremiereDate);
+            Assert.NotEmpty(res.People);
+            Assert.Equal("Krysta Youngs", res.People[0].Name);
+            Assert.Equal(PersonType.Composer, res.People[0].Type);
+            Assert.Equal("Julia Ross", res.People[1].Name);
+            Assert.Equal(PersonType.Composer, res.People[1].Type);
+            Assert.Equal("Yiwoomin", res.People[2].Name);
+            Assert.Equal(PersonType.Composer, res.People[2].Type);
+            Assert.Equal("Ji-hyo Park", res.People[3].Name);
+            Assert.Equal(PersonType.Lyricist, res.People[3].Type);
+            Assert.NotEmpty(res.Genres);
```

Comment:
Don't need this bc of the next line

Reply:
How so? I thought that `Assert.Same` was testing for identical instances, while `Assert.Equal` checked the contents?

Should I assert each value like above? Is there a neater way to do that? (Both for the genres array and the persons one)

Reply:
Right, I see that `Assert.Equal` uses the Equal method of the type, which arrays don't override, so it checks for instance too.